airflow/www/app.py

```diff
@@ -131,7 +132,29 @@ def create_app(config=None, testing=False):
 
     init_robots(flask_app)
 
+    # Configure caching
+    webserver_caching_hash_method = conf.get(section="webserver", key="CACHING_HASH_METHOD", fallback=None)
     cache_config = {"CACHE_TYPE": "flask_caching.backends.filesystem", "CACHE_DIR": gettempdir()}
```

Comment:
You need to add this new parameter in `airflow/config_templates/config.yml` with description and defaults and let pre-commit do the job of updating the documentation.

Reply:
Ah, great, thanks! Addressing the other comments now 👍